src/components/layout/AppLayout.tsx

```diff
@@ -53,12 +53,21 @@ const styles = (theme: { workspaces: { drawer: { width: any } } }) => ({
     // width: `calc(100% + ${theme.workspaces.drawer.width}px)`,
     width: '100%',
     transition,
+  },
+  appContentTransformTextWorkspace: {
     transform() {
       return workspaceStore.isWorkspaceDrawerOpen
```

Comment:
Would it be better to move the ternary to be evaluated inside the `translateX()` like so?
```
return 'translateX(`workspaceStore.isWorkspaceDrawerOpen ? 0 : -75px`)');
```
(Syntax might be incorrect since I'm not typing inside an editor)

Reply:
Sure, we can do that